# StateActionPair.msg
string state
int32 action